docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,122 @@
+    * [Choosing a batch processing technology in Azure](/azure/architecture/data-guide/technology-choices/batch-processing)
+    * [Choosing an analytical data store in Azure](/azure/architecture/data-guide/technology-choices/analytical-data-stores)
+    * [Choosing a data analytics technology in Azure](/azure/architecture/data-guide/technology-choices/analysis-visualizations-reporting)
+
+## Considerations
+
+### Availability
+
+SQL Data Warehouse has [guaranteed SLAs](http://azure.microsoft.com/support/legal/sla/sql-data-warehouse/v1_0/) and [recommended practices for achieving high availability](http://azure/sql-data-warehouse/sql-data-warehouse-best-practices).
+
+Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+
+For other availability topics, see the [availability checklist][availability] in the Azure Architecure Center.
+
+### Scalability
+
+The [massively parallel processing architecture](/azure/sql-data-warehouse/massively-parallel-processing-mpp-architecture) of SQL Data Warehouse provides scalability and high performance. [SQL Data Warehouse Gen2](/azure/sql-data-warehouse/memory-and-concurrency-limits) offers the greatest level of scale (up to 30,000 Data Warehouse Units) and providing unlimited columnar storage. You can also [scale SQL Data Warehouse on demand](/azure/sql-data-warehouse/sql-data-warehouse-manage-compute-overview), reducing or even pausing compute during periods of low demand or inactivity to lower your overall costs.
+
+Azure Analysis Services can be [configured for scale-out](/azure/analysis-services/analysis-services-scale-out). With scale-out, client queries can be distributed among multiple query replicas in a query pool, reducing response times during high query workloads. You can also separate processing from the query pool, ensuring client queries are not adversely affected by processing operations. 
+
+For other scalability topics, see the [scalability checklist][scalability] in the Azure Architecure Center.
+
+### Security
+
+The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+For general guidance on designing secure solutions, see the [Azure Security Documentation][security].
+
+### Resiliency
+
+For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
```

Comment:
If we don't have anything specific to say about resiliency, I think we should pull it out.  Eventually I'd like to talk about where we're using managed services, where single points of failure are, and what you need to think about to keep this up and running.